docs/system_overview/configuration.md

```diff
@@ -193,8 +193,12 @@ This file defines the behavior of the server Agent that answers individual reque
         // ... Additional Entries ...
     ],
 
-    // The fully qualified domain name or hostname which the server identifies as.
-    // The name defined by this setting must be resolvable by the server.
+    // The FQDN or hostname which identifies the server.
+    //
+    // The following rules apply:
+    // - Must not be localhost
```

Comment:
Would it be clearer with quotation marks?
```suggestion
    // - Must not be the string "localhost"
```

Reply:
Agree

Reply:
Will update.

Reply:
Done.